drivers/interrupt_controller/intc_ite_it8xxx2.c

```diff
@@ -69,6 +69,131 @@ inline void set_csr(unsigned long bit)
 	}
 }
 
+#ifdef CONFIG_SOC_IT8XXX2_PLL_FLASH_48M
```

Comment:
Why is this code added to the INTC driver?  Creating clock_control_ite_it8xxx2.c is better.  Alternatively, this might make sense to add to riscv-ite/it8xxx2/soc.c.

Reply:
Done. move code into soc.c